src/main/java/ui/Ui.java

```diff
@@ -33,7 +33,7 @@ public void showWelcome() {
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println(logo);
         showLine();
-        print("Hello! I'm Duke\nWhat can I do for you?");
+        print("Hello! I'm duke.Duke\nWhat can I do for you?");
```

Comment:
Couldn't help notice that maybe when you refactored duke into a package, it also refactored all your comments and messages.

Reply:
thanks for reminding me! I didn't notice it. Any good methods to avoid this situation when refactoring code?